pkg/aws/aws.go

```diff
@@ -121,6 +121,16 @@ func CheckInstanceIsRunning(ec2Svc *ec2.EC2, nodeID string) (bool, error) {
 	return false, nil
 }
 
+func StopInstanceWoEIP(ec2Svc *ec2.EC2, instanceID string) error {
```

Comment:
can we add a bool to StopIntance to avoid releasing ip? so as to have one function only